benchbuild/utils/run.py

```diff
@@ -5,14 +5,20 @@
 from contextlib import contextmanager
 
 import attr
+import plumbum as pb
```

Comment:
Codacy found an issue: [Unable to import 'plumbum'](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)